tests/e2e/auth/cred.go

```diff
@@ -88,22 +90,63 @@ var _ = Describe("Azure Credential Provider", Label(utils.TestSuiteLabelCredenti
 		err = utils.DockerLogout()
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	It("should be able to create an ACR cache and pull images from it", Label(utils.TestSuiteLabelOOTCredential), func() {
```

Comment:
what do you mean `create an ACR cache`?

Reply:
I am using a preview feature of ACR: [acr cache](https://learn.microsoft.com/en-us/azure/container-registry/tutorial-registry-cache)
The reason is that I want to test Windows but control-plane Node is Linux. So, it is hard to push a Windows image to ACR. With this new feature, I can skip the pushing.

Reply:
thanks, pls add this comment into the test suites, that would make it more clear.

Reply:
Makes sense. Updated.